src/sentry/issues/escalating.py

```diff
@@ -258,14 +258,16 @@ def get_group_hourly_count(group: Group) -> int:
     return int(hourly_count)
 
 
-def is_escalating(group: Group) -> bool:
-    """Return boolean depending on if the group is escalating or not"""
+def is_escalating(group: Group) -> Tuple[bool, Optional[int]]:
+    """
+    Return boolean depending on if the group is escalating or not and the forecast if it exists.
```

Comment:
We can skip mentioning the return type since the function is typed clearly

```suggestion
    Return whether the group is escalating and the forecast if it exists.
```